### PR TITLE
Pivotal ID # 184364448: Previos version file check taking longer than expected

### PR DESCRIPTION
**Pull request by Juan-EBI**
https://www.pivotaltracker.com/story/show/184364448

### CODE REVIEW

File: submission/persistence-mongo/src/main/kotlin/ac/uk/ebi/biostd/persistence/doc/migrations/DatabaseChangeLog.kt [hunk 1]
```diff
@@ -187,3 +189,18 @@ class ChangeLog007 {
         }
     }
 }
+
+@ChangeLog
+class ChangeLog008 {
+    @ChangeSet(order = "008", id = "File List files index", author = "System")
+    fun changeSet007(template: MongockTemplate) {
+        template.indexOps(FileListDocFile::class.java).apply {
```
Comment (jhoanmanuelms) [inline submission/persistence-mongo/src/main/kotlin/ac/uk/ebi/biostd/persistence/doc/migrations/DatabaseChangeLog.kt:197]: 👍

File: submission/submission-webapp/src/main/resources/application.yml [hunk 2]
```diff
@@ -1,3 +1,5 @@
+
```
Comment (jhoanmanuelms) [inline submission/submission-webapp/src/main/resources/application.yml:1]: Why did you add these two lines? Is it a style guideline?

**Review by jhoanmanuelms — APPROVED** [review]
LGTM. One minor question

File: submission/submission-webapp/src/main/resources/application.yml (hunk 2 above)
Comment (Juan-EBI) [reply submission/submission-webapp/src/main/resources/application.yml:1]: mistake
